week1/homework/src/server.js

```diff
@@ -5,11 +5,43 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+let responseObject = {'state': 10};
+const responseError = { "error": "Not found"};
+const urls = ['/state', '/add', '/subtract', '/reset'];
 function createServer(port) {
-  let state = 10;
-
+  let responseObject = {'state': 10};
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    const status = (urls.some(elm => elm === request.url))? 200 : 404;
+    response.writeHead(status, {"Content-type": "application/json"})
+    switch(request.url)
+  	{        
+      case '/add':
+      {
```

Comment:
i know that i dont need the { } for a switch case, but i was having  a problem in the begin, and i decided to try the bracket just to verify if my code was good indented.
at the end the error was that the test is concurrent so i need to create the response object inside the method, instead of having it global

Reply:
OK, so now you know what the problem was, so you can remove the curly braces...